app/models/normal_hour.rb

```diff
@@ -34,45 +34,57 @@ def get_resource
 
   # hr_start_time
   # @author David J. Davis
-  # @return a human readable time string 
+  # @return a human readable time string
   def hr_open_time
     human_readable_time(open_time)
   end
 
   # hr_close_time
   # @author David J. Davis
-  # @return a human readable time string 
+  # @return a human readable time string
   def hr_close_time
     human_readable_time(close_time)
   end
 
   # weekday
   # @author David J. Davis
-  # @returns the day of the week Monday, Tuesday, Wed etc. 
+  # @returns the day of the week Monday, Tuesday, Wed etc.
   def weekday
     day = day_of_week.to_i
     Date::DAYNAMES[day]
   end
 
+  # open_close_time
+  # @author Tracy A. McCormick
+  # @param time a time string from the database
+  # @return string containing both the hr_open_time and hr_end_time
+  def open_close_time
+    hr_open_time + ' - ' + hr_close_time if !open_time.nil? && !close_time.nil?
+  end
+
+  # comment
+  # @author Tracy A. McCormick
+  def comment
```

Comment:
I think we should find a better method for representing this.  

comment_string =  (open_time.nil? && close_time.nil?) ? 'Closed' :  ''